app.json

```diff
@@ -7,9 +7,18 @@
     "githubUrl": "https://github.com/smart-village-solutions/smart-village-app-app",
     "platforms": ["ios", "android"],
     "version": "4.1.0",
-    "runtimeVersion": { "policy": "appVersion" },
-    "extra": { "eas": { "projectId": "20540f3a-6e82-4038-8987-adb6d97f0e75" } },
-    "updates": { "url": "https://u.expo.dev/20540f3a-6e82-4038-8987-adb6d97f0e75" },
+    "otaVersion": 1,
```

Comment:
It looks very good. If we run `yarn update:all`, the otaVersion will be increased automatically, right? We don't need to increase this value extra. Well I think we will commit the updated otaVersion later.

Reply:
I do not understand? The script increments the `otaVersion` as i wrote in the PR description. And Git actions are performed as well, see the last 7 lines of the new script.